example/C/src/ContainerizedFederatedHelloWorld/README.md

```diff
@@ -0,0 +1,51 @@
+# ContainerizedFederatedHelloWorld
+This is a basic example showing containerized federated execution. 
+
+For more details, see: https://github.com/icyphy/lingua-franca/wiki/
```

Comment:
This looks great!

I suggest that if you want to link to a specific page, you could link to [Distributed Execution](https://github.com/lf-lang/lingua-franca/wiki/Distributed-Execution). We should probably add a docker section there.

Reply:
wait... that was a mistake. I meant to link to https://github.com/lf-lang/lingua-franca/wiki/Containerized-Execution

Reply:
Ah okay, that one is even better.